test/OpenSpaceToolkit/Physics/Environment.test.cpp

```diff
@@ -753,3 +757,144 @@ TEST (OpenSpaceToolkit_Physics_Environment, Test_3)
 }
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+TEST (OpenSpaceToolkit_Physics_Environment, EarthCreationTest)
```

Comment:
rename test: We are testing the creation of an `Environment` using Earth Model only but this function should focus on ensuring the creator for `Environment` is functional!